src/zope/index/text/ricecode.py

```diff
@@ -14,7 +14,7 @@
 """Rice coding (a variation of Golomb coding)
 
 Based on a Java implementation by Glen McCluskey described in a Usenix
- ;login: article at
+article at
```

Comment:
AFAIU *;login:* is the name of the magazine published by Usenix: https://www.usenix.org/publications/login

Reply:
That text triggered a Sphinx warning ("docstring of zope.index.text.ricecode:5: WARNING: Definition list ends without a blank line; unexpected unindent.") (and was jarring to read). Rather than try to figure out why, it was easier to just remove it.